docs/source/user-guide/vision/tutorials/plot_custom_task_tutorial.py

```diff
@@ -164,142 +152,115 @@ def load_or_download(cls, root: Path, train: bool) -> 'CocoInstanceSegmentationD
 
 
 # Download and load the datasets
-curr_dir = Path('.')
-train_ds = CocoInstanceSegmentationDataset.load_or_download(curr_dir, train=True)
-test_ds = CocoInstanceSegmentationDataset.load_or_download(curr_dir, train=False)
-
-
-def batch_collate(batch):
-    """Function which gets list of samples from `CocoInstanceSegmentationDataset` and combine them to a batch."""
-    images, classes, masks = zip(*batch)
-    return list(images), list(classes), list(masks)
-
-
-# Create DataLoaders
-train_data_loader = DataLoader(
-    dataset=train_ds,
-    batch_size=32,
-    shuffle=False,
-    collate_fn=batch_collate
-)
-
-test_data_loader = DataLoader(
-    dataset=test_ds,
-    batch_size=32,
-    shuffle=False,
-    collate_fn=batch_collate
-)
+train_ds = CocoInstanceSegmentationDataset.load_or_download(Path('.'), train=True)
+test_ds = CocoInstanceSegmentationDataset.load_or_download(Path('.'), train=False)
 
 # %%
 # Visualizing that we loaded our datasets correctly:
 
-masked_images = [draw_segmentation_masks(train_ds[i][0], masks=train_ds[i][2], alpha=0.7)
-                 for i in range(5)]
+masked_images = [draw_segmentation_masks(train_ds[i][0], masks=train_ds[i][1], alpha=0.7) for i in range(5)]
 
-fix, axs = plt.subplots(ncols=len(masked_images), figsize=(20, 20))
+fig, axs = plt.subplots(ncols=len(masked_images), figsize=(20, 20))
 for i, img in enumerate(masked_images):
     img = img.detach()
     img = F.to_pil_image(img)
     axs[i].imshow(np.asarray(img))
     axs[i].set(xticklabels=[], yticklabels=[], xticks=[], yticks=[])
 
-fix.show()
+fig.show()
 
 # %%
-# Implement Custom Task
-# =====================
-# With our data and model ready we can write the task class.
+# Implementing the VisionData class
+# =================================
+# The checks in the package validate the data by calculating various quantities over the data, labels and
+# predictions (when available). In order to do that, those must be in a pre-defined format, according to the task type.
+#
+# In the following example we're using pytorch. To see how this can be done using tensorflow or a generic generator,
+# please refer to :ref:`creating VisionData guide <vision_data__creating_vision_data>`.
```

Comment:
```suggestion
# please refer to :ref:`creating VisionData guide <vision_data__creating_vision_data>`.
```

I think I referred to this label as possibly irrelevant but then I see it's used here...
so I I guess either leave the label for reference as is, or alternatively to replace the reference with a :doc: link to the whole page and not specifically that section.

Reply:
(and I think it was already deleted in a previous commit so either return it or change here...)